src/uct/ib/base/ib_iface.c

```diff
@@ -299,7 +304,7 @@ void uct_ib_iface_address_pack(uct_ib_iface_t *iface, const union ibv_gid *gid,
                                uint16_t lid, uct_ib_address_t *ib_addr)
 {
     uct_ib_address_pack(gid, lid, uct_ib_iface_is_roce(iface),
-                        iface->is_global_addr, ib_addr);
+                        1, ib_addr);
```

Comment:
it means we always pack interface id? and increase address size?
why need to change it?

Reply:
fixed. added evaluation for link_local